modular_skyrat/modules/medical/code/anesthetic_machine.dm

```diff
@@ -39,7 +39,7 @@
 		return TRUE
 
 	if(attached_tank)
-		to_chat(user, span_warning("[attached_tank] inside of the [src] deconsturction."))
+		to_chat(user, span_warning("\The [attached_tank] must be removed from \the [src] first!"))
```

Comment:
adding \the does nothing, remove them
![image](https://github.com/Skyrat-SS13/Skyrat-tg/assets/41448081/8ddf0a6e-2d67-4cb7-ae49-b6eb6fb3c877)

Reply:
Didn't you just suggest changes on another PR to add that...? I mean, I know it doesn't do anything, but I find that using `\the` and `\The` is easier to read and more explicit. I can change it, though.

Reply:
P sure i suggested a removal